ui/blocks/LanguageSwitcher.tsx

```diff
@@ -25,7 +27,10 @@ const LanguageSwitcher: React.FC = () => {
   const handleItemClick = useCallback(
     (ev: React.SyntheticEvent<HTMLAnchorElement>) => {
       const { locale } = ev.currentTarget.dataset
+
       persistLocale(locale)
+
+      setTimeout(() => router.reload(), 300)
```

Comment:
See we _do_ have to reload a page when changing the locale. Somehow client side there is some issues with state for queries. Couldn't find out why the query is empty (see bug report #326)